yalafi/parameters.py

```diff
@@ -63,6 +63,7 @@ def init_macros(self):
 }
         \newcommand{\qquad}{\;}
         \newcommand{\quad}{\;}
+        \newcommand{\nobreakspace}{~}

```

Comment:
Hesitating to ask: Could you please shift that to below the definition of `\newline`? And could you add a test line in `tests/test_packages/test_latex_builtins.py`, below line 34?